src/components/utils/src/file_system_qt.cc

```diff
@@ -266,13 +261,11 @@ const std::string file_system::ConvertPathForURL(const std::string& utf8_path) {
 }
 
 bool file_system::CreateFile(const std::string& utf8_path) {
-  std::ofstream file(QString(utf8_path.c_str()).toStdWString());
-  if (!(file.is_open())) {
+  QFile file(QString(utf8_path.c_str()));
+  if (!file.open(QIODevice::WriteOnly)) {
     return false;
```

Comment:
`if ' is redundant. QFile::open - returned bool
You can use `return flle.open(..)`